components/insight/SRC/org/openmicroscopy/shoola/util/VersionCompare.java

```diff
@@ -0,0 +1,36 @@
+package org.openmicroscopy.shoola.util;
+
+public class VersionCompare {
+
+    /**
+     * Compares two version strings. https://stackoverflow.com/a/6702029/272446
+     * <p>
+     * Use this instead of String.compareTo() for a non-lexicographical
+     * comparison that works for version strings. e.g. "1.10".compareTo("1.6").
+     *
+     * @param str1 a string of ordinal numbers separated by decimal points.
+     * @param str2 a string of ordinal numbers separated by decimal points.
+     * @return The result is a negative integer if str1 is _numerically_ less than str2.
+     * The result is a positive integer if str1 is _numerically_ greater than str2.
+     * The result is zero if the strings are _numerically_ equal.
+     * @note It does not work if "1.10" is supposed to be equal to "1.10.0".
```

Comment:
This is failing javadoc: https://ci.openmicroscopy.org/job/OMERO-DEV-merge-build/ICE=3.6,jdk=8_LATEST,label=octopus/1064/warnings42Result/source.6931003519486980497/#8